authguard.service

```diff
@@ -7,6 +7,15 @@ Wants=network-online.target
 Type=simple
 ExecStart=/opt/authguard -web.listen-address=:8080 -web.proxy-to=127.0.0.1:9090 -user=username -pass=password -crt=/etc/ssl/private/tmpcert.pem -key=/etc/ssl/private/tmpkey.pem
 Restart=always
+
+# optional systemd hardening-options Documentation about these can be found under systemd.exec(5)
+
```

Comment:
newline behind "hardening-options" please and no empty line here to keep things compact.